infrastructure/haproxy_reload.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
-#:80
-real=0
+#:80/443
+http=0
 #:81
 stats=1
 
+DPORT=80
+[ ${HAPROXY_SSL} == "true" ] && DPORT=443 && export CERT_OPTS="${HAPROXY_CERT_OPTS}"
```

Comment:
you don;t need to export it if you use HAPROXY_CERT_OPTS in infrastructure/haproxy.cfg